### PR TITLE
LPS-167882 Don't use aui:input to avoid field validation

### PR DESCRIPTION
**Pull request by sandrodw3**
Motivation
=======
This is a follow-up of https://github.com/liferay-echo/liferay-portal/pull/11171. After testing it, the QA told us that the Cancel button was not being well read by the screen reader, as it announces the field error when focusing it.
The fix recommendation was removing the field validation, as we already have the alert for it.

Proposed Solution
========
What I have done is removing the use of `aui:input` and adding the markup manually, so we avoid the field validation. I'm not super convinced of this fix but it was the only idea that came to my mind.

Steps to Verify
========
1. Go to Site Builder > Pages
2. Click + button
3. Click on Add button without filling a name
4. See how field validation doesn't appear 
5. Focus Cancel button and see how the screen reader reads it properly

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by sandrodw3** [comment]
ci:test:sf

File: modules/apps/layout/layout-admin-web/src/main/resources/META-INF/resources/add_layout.jsp [hunk 1]
```diff
@@ -31,7 +31,21 @@ List<SiteNavigationMenu> autoSiteNavigationMenus = layoutsAdminDisplayContext.ge
 		onSubmit="event.preventDefault();"
 	>
 		<liferay-frontend:edit-form-body>
-			<aui:input label="name" name="name" placeholder='<%= LanguageUtil.get(request, "add-page-name") %>' required="<%= true %>" />
+			<label class="control-label" for="<portlet:namespace />name">
```
Comment (victorg1991) [inline modules/apps/layout/layout-admin-web/src/main/resources/META-INF/resources/add_layout.jsp:34]: I believe we can just set `validateOnBlur="<%= false %>"` on the edit-form tag

Comment (victorg1991) [reply modules/apps/layout/layout-admin-web/src/main/resources/META-INF/resources/add_layout.jsp:34]: Could you try it @sandrodw3 ? 😇

Comment (sandrodw3) [reply modules/apps/layout/layout-admin-web/src/main/resources/META-INF/resources/add_layout.jsp:34]: Weee. I've tried it but it only works for not validating on blur, when trying to submit it validates 😞 what do you think?

**Comment by victorg1991** [comment]
ci:forward

**Comment by ealonso** [comment]
ci:forward

**Comment by ealonso** [comment]
ci:stop

**Comment by victorg1991** [comment]
ci:forward

**Comment by ealonso** [comment]
Hey @victorg1991 @sandrodw3, Could you please resend? the tests never ends

**Pull request closed without merging**